omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -41,6 +41,9 @@
 
 default['private_chef']['fips_enabled'] = ChefConfig.fips?
 
+default['private_chef']['required_recipe']['enable'] = false
```

Comment:
Circling back on this, I definitely think sticking with both is ideal. If we ever move to a place where this file is stored in a database then we're unlikely to have a `path`. In that scenario we'd need would be left without a toggle.